settingtypes.txt

```diff
@@ -46,3 +46,12 @@ share_bones_time (Bone share time) int 1200 0
 
 #    Replaces old stairs with new ones. Only required for older worlds.
 enable_stairs_replace_abm (Replace old stairs) bool false
+
+
+#    If enabled, the reason why bones where dropped or not dropped, along
+#    with death location, is written to the minetest log.
+enable_bones_logging (Log location of deaths and reason if no bones) bool true
+
+#    If enabled, the reason why bones where dropped or not dropped, along
+#    with death location, is sent as a chat message to player.
+enable_bones_chat_msg (Send player last death location and reason if no bones) bool false
```

Comment:
These short descriptions should be shorter. How about: `Inform player about bones`?